spec/person_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../person'
+
+RSpec.describe Person do
+  before :each do
+    @person = Person.new(12, parent_permission: true, name: 'Ichsan')
+  end
+
+  describe '#age' do
+    it 'should be 12' do
+      expect(@person.age).to eql(12)
+    end
+  end
+
+  describe '#parent_permission' do
+    it 'should be true' do
+      expect(@person.parent_permission).to eql(true)
+    end
+  end
+
+  describe '#name' do
+    it 'should be ichsan' do
+      expect(@person.name).to eql('Ichsan')
+    end
+  end
+end
```

Comment:
- [ ] Hi @kamranasif506, BASED on the requirements of this project, there should be tests for all the methods in all the classes. Hence you'll required to test for the #**can_use_service?** method(Both positive and negative scenarios), #**correct_name** method that corrects the name, adding a new rental method that returns rental and adds its to the book.
- [ ] It'll also be nice to kindly test the case where  the constructor method sets default value for the optional name parameter to "Unknown" when the user doesn't provide the name :+1: 
- [ ] Kindly check out also best testing practices here. Instead of using **describe** block so many times in a test file, you can instead make use of **context** block that wrap the "it" blocks.
[Hints](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/rspec_unit_test.md#unit-testing)